src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -365,8 +370,10 @@ internal void StartProcess(string workerId, Process process)
                     HandleWorkerError(new Exception("Worker process is not attached"));
                 }
             };
-            _logger.LogInformation($"Start Process: {process.StartInfo.FileName} {process.StartInfo.Arguments}");
+            _logger.LogInformation($"Starting {process.StartInfo.FileName} language worker process with Arguments={process.StartInfo.Arguments}");
             process.Start();
+            _logger.LogInformation($"{process.StartInfo.FileName} process with Id={process.Id} started");
+            _processRegistry?.Register(_process);
```

Comment:
This call should probably me moved to `StartWorker`, so the scope of `StartProcess` remains the same (it only starts the process with the appropriate configuration)